app/forms/project_form.rb

```diff
@@ -68,7 +68,7 @@ def invite_to_github_team
     github.invite_to_team(team_id, username)
   end
 
-  def create_and_fork_repo
+  def create_and_clone_repo
```

Comment:
Assignment Branch Condition size for create_and_clone_repo is too high. [15.13/15]